src/components/NewLexemeForm.vue

```diff
@@ -46,20 +46,27 @@ export default {
 		<lexical-category-input
 			v-model="lexicalCategory"
 		/>
-		<wikit-button
-			class="form-button-submit"
-			type="progressive"
-			variant="primary"
-		>
-			{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
-		</wikit-button>
+		<div>
+			<wikit-button
+				class="form-button-submit"
+				type="progressive"
+				variant="primary"
+			>
+				{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
+			</wikit-button>
+		</div>
 	</form>
 </template>
 
 <style scoped lang="scss">
 @import "@wmde/wikit-tokens/variables";
 
 .wbl-snl-form {
+	// display
+	display: flex;
+	flex-direction: column;
+	gap: $dimension-layout-xsmall;
```

Comment:
Hm, [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/gap#browser_compatibility) claims that `gap` support in Flex Layout is actually really recent? Chrome 84, Firefox 63, Safari 14.1…

Maybe there’s an older, flex-specific property we should use instead?

Reply:
:sob:

Reply:
Now it should work everywhere, ugly as it is